crypto/util.go

```diff
@@ -85,8 +85,14 @@ func Hash(data []byte) Digest {
 	return sha512.Sum512_256(data)
 }
 
+// HashObjSlow computes a hash of a Hashable object and its type, doing so the
```

Comment:
`HashObjSlow` appears to be only used in genesis_test.go so it can be moved there?

Reply:
Yeah, since I've only retained that one benchmark, I've moved it.